src/test/scala/Test.scala

```diff
@@ -1,15 +1,22 @@
 import java.net.URLClassLoader
-import java.nio.file.{ Paths, Files }
+import java.nio.file.{Files, Path, Paths}
+
 import net.kaliber.pdf.PdfRenderer
 import org.qirx.littlespec.Specification
+import org.xhtmlrenderer.pdf.ITextRenderer
+
 import scala.io.Source
 
 object Test extends Specification {
 
-  val examples = Paths.get("./example")
-  val `test.pdf` = examples resolve "test.pdf"
+  val examples: Path = Paths.get("./example")
+  val `test.pdf`: Path = examples resolve "test.pdf"
+
+  // Font should be rendered from anywhere
+  val rendererWithFonts:ITextRenderer = new ITextRenderer
+  rendererWithFonts.getFontResolver.addFontDirectory(examples.toAbsolutePath.toString + "/fonts", true)
 
-s"""|============================
+  s"""|============================
```

Comment:
If you indent this line, you should also indent the ones below. Make sure the `|` chars line out. I do not think the indentation is an improvement, but that's taste. So do with it what you want 😄